scripts/apps/monitoring/directives/ArticlesListReact.tsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/no-multi-comp */
+
+import React from 'react';
+import {openArticle} from 'core/get-superdesk-api-implementation';
+import {ISuperdeskQuery} from 'core/query-formatting';
+import {dataApi} from 'core/helpers/CrudManager';
+import ng from 'core/services/ng';
+import {IArticle, IDesk, IUser} from 'superdesk-api';
+import {ArticlesListByQueryWithFilters} from 'core/ArticlesListByQueryWithFilters';
+
+interface IProps {
+    heading: string;
+    getQuery(userId: string, deskId: string): ISuperdeskQuery;
```

Comment:
not sure about those params here, imo the component shouldn't be passing anything to `getQuery` method but rather the parent component which knows what is needed there.